src/substrate/filters/enricher.ts

```diff
@@ -106,15 +106,14 @@ export async function Enrich(
         const validators = await api.derive.staking.validators();
         const electedInfo: DeriveStakingElected = await api.derive.staking.electedInfo() as any; // validator preferences for getting commision
         const validatorEraPoints: EraRewardPoints = await api.query.staking.currentPoints() as EraRewardPoints;
-        const currentEra = (Number)(await api.query.staking.currentEra<Option<EraIndex>>());
+        const currentEra = (Number)(await (await api.query.staking.currentEra<Option<EraIndex>>()).unwrap());
```

Comment:
why two awaits? I've never seen this before.

Reply:
Ah, my VSCode extension added it, it has no effect, must have missed it, thanks for pointing out, I'd remove it!

Reply:
apparently tests are failing without the second await, unwrap feels like needing await, I don't think this extra await would be a problem.